packages/utils/src/bulk-query-patients-simplified.ts

```diff
@@ -97,12 +97,19 @@ async function queryDocsForPatient(
   log: typeof console.log
 ) {
   try {
-    const docQueryPromise = async () =>
-      triggerDocQuery(cxId, patientId, triggerWHNotificationsToCx);
     const getPatientPromise = async () => metriportAPI.getPatient(patientId);
+    const patient = await getPatientPromise();
+
+    if (!patient.facilityIds || patient.facilityIds.length === 0) {
+      log(`Patient ${patientId} has no facilities, skipping...`);
+      return;
+    }
+
+    const facilityId = patient.facilityIds[0];
```

Comment:
I know these are just utils, but using the right NPI _whenever possible_ still applies.

Maybe we can add a new const after the main TSDoc at the top of the file, and indicate it _should_ be populated before using the script?

Then, here, we'd try to use that ID set at the top, but if it's not set then we revert to using the first facilityId on the pt's array.

Here and other utils, with the exception of `enhance-coverage.ts` which is not used anymore - I'll ticket removing it if not exists yet.

Reply:
Done - this also seems like the only script that has it besides enhance-coverage